examples/example_app/example_app/jobs.py

```diff
@@ -186,6 +187,47 @@ def receive_job_button(self, obj):
             self.logger.error("Unable to run Job Button for type %s.", type(obj).__name__, extra={"object": obj})
 
 
+class WorkListJob(Job):
+    """Job that monitors the list of jobs."""
+
+    class Meta:
+        """Meta object boilerplate for receptorctl."""
+
+        name = "Perform ReceptorCtl Work List"
+        description = "Run ReceptorCtl Work List"
+        has_sensitive_variables = False
+
+    def run(self, *args, **data):  # pylint: disable=unused-argument
+        """Run receptorctl command."""
+        self.logger.warning("Run ReceptorCtl Work List.")
+        r = ReceptorControl("tcp://node01.example.internal:1234")
```

Comment:
This should be changed to 
```suggestion
        r = ReceptorControl("tcp://remote-worker.example.internal:1234")
```